tests/trainer/test_trainer.py

```diff
@@ -2059,7 +2059,6 @@ def training_step(self, batch, batch_idx):
         ({"strategy": None}, SingleDeviceStrategy, "single_device", CPUAccelerator, 0),
         ({"strategy": "dp"}, DDPStrategy, "ddp", CPUAccelerator, 0),
         ({"strategy": "ddp"}, DDPStrategy, "ddp", CPUAccelerator, 0),
-        ({"strategy": "ddp", "num_processes": 2}, DDPStrategy, "ddp", CPUAccelerator, 0),
```

Comment:
This test case is the same with L2075, remove duplicated test case. 
https://github.com/PyTorchLightning/pytorch-lightning/blob/7e2f9fbad555242b0ceb2a24e5e4c004f0701bae/tests/trainer/test_trainer.py#L2075